src/parser/parseTransformAttribute.ts

```diff
@@ -110,13 +55,14 @@ export function parseTransformAttribute(attributeValue) {
     return matrix;
   }
 
-  attributeValue.replace(reTransform, function (match) {
-    const m = new RegExp(transform).exec(match).filter(function (match) {
-        // match !== '' && match != null
-        return !!match;
-      }),
-      operation = m[1],
-      args = m.slice(2).map(parseFloat);
+  for (const match of attributeValue.matchAll(reTransform)) {
+    const transformMatch = new RegExp(transform).exec(match[0]);
```

Comment:
i never remember how regexp work, in this case can we create the regexp before the loop and just use it for each match[0]?

Reply:
Dido.
It never does what you want it to.
And mdn says one thing while chrome does another from what I remember